build.sh

```diff
@@ -74,11 +103,14 @@ for PKG_PATH in $(catkin_topological_order --only-folders); do
 
   # dpkg-source-opts: no need for upstream.tar.gz
   sbuild --chroot-mode=unshare --no-clean-source --no-run-lintian \
-    --dpkg-source-opts="-Zgzip -z1 --format=1.0 -sn" --build-dir=/home/runner/apt_repo \
-    --extra-package=/home/runner/apt_repo "$@"
+    --dpkg-source-opts="-Zgzip -z1 --format=1.0 -sn" --build-dir=${APT_REPO} \
+    --extra-package=${APT_REPO} "$@"
   )
   COUNT=$((COUNT+1))
   echo "::endgroup::"
 done
 
+# revert rosdep cache
```

Comment:
dito this.

Reply:
Same question as https://github.com/jspricke/ros-deb-builder-action/pull/10#discussion_r1101709187

PS: it's ditto, with two "t" ;-)

Reply:
Interesting, it is dito or detto in German and French (according to Wikipedia) but ditto in Italian and English, thx :).

Reply:
Ah, the original is Italian (because `dito` means finger), but I guess German wouldn't have that problem so I presume that's why `dito` is correct. Interesting there's a `detto` variant.